test/tutorial/scripts/cli/delete_bundle_cli.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+# CANNOT TEST THIS
```

Comment:
Understood.  Please remove this file.  This doesn't need to be tested or be put into the docs for the time being.